lib/metrics/registry_test.go

```diff
@@ -0,0 +1,53 @@
+package metrics
```

Comment:
We really should add a linter for this :smile: Maybe [go-header](https://github.com/denis-tingaikin/go-header) works now?

```suggestion
/*
 *
 * k6 - a next-generation load testing tool
 * Copyright (C) 2021 Load Impact
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Affero General Public License as
 * published by the Free Software Foundation, either version 3 of the
 * License, or (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU Affero General Public License for more details.
 *
 * You should have received a copy of the GNU Affero General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 *
 */

package metrics
```

Reply:
uff

Reply:
unfortunately it seems like there has been no improvement on the areas that were previously problematic from a quick look :(

Reply:
I found other files with it missing so I prefer to close it in another PR with all of them